static/src/javascripts/projects/common/modules/commercial/sticky-ad-banner.js

```diff
@@ -124,11 +124,12 @@ define([
         var transitionTimingFunction = 'cubic-bezier(0, 0, 0, .985)';
         var transitionDuration = '1s';
 
-        els.$header.css({
+        els.$page.css({
+            'will-change': 'transform',
```

Comment:
> As for removing it, why not wait for the transitionend event?

Yeah, the `AD_BANNER_TRANSITION_END` action :)

>  I'm not sure one animation frame is enough to let the browser do whatever groundwork it can do

I don't know, I reckon it might be. Isn't the point with `will-change` that it is a just-in-time tool that should be attached to tell the browser exactly when it should be performing it's performance optimisations. [The spec](https://www.w3.org/TR/css-will-change-1/#dont-waste) says you should always let the browser handle things as normal if you're not expecting this change to happen soon.